contents/blog/the-posthog-array-1-37-0.mdx

```diff
@@ -0,0 +1,146 @@
+* **Completed an event regularly:** Find your power users. For example, "Give me users who bought an item in 5 out of the last 7 weeks".
+* **Stopped doing an event:** Find your users that are at risk of churning. For example, "Give me users who haven't bought anything in the last 7 days, but had bought something in the 30 days prior".
+* **Started doing an event again:** Find your users that did not churn. For example, "Give me users who bought something, then didn't for 2 weeks, but have again in the last 7 days".
+
+#### AND/OR operators
+
+You can now combine these new conditions using complex AND/OR groupings.
+
+For example, you can define a cohort that is "Give me users who are outside of Europe AND have either regularly bought an item 4 of the last 5 weeks OR bought 3 items in the past week".
+
+With the combination of these new conditions and the new AND/OR operators, you can now define extremely powerful cohorts for finding the exact users that you're looking for. 
+
+#### Nested cohorts
+
+But that's not all! You can also use existing cohorts to define _new_ cohorts. For example, you can define a cohort as "Give me users in _Cohort X_ that are not in _Cohort Y_".
+
+Using these nested cohorts enables you to avoid redefining the same cohort over and over again.
+
+### New: Event and property detail pages
+
+We've added a new page within Data Management, which enables you to dig into the details of all events and properties. For example, on the event page, you can now see:
+* When an event was first and last seen
+* How many times it was sent in the last 30 days
+* The top properties for that event
+* A filterable list of the specific event for you to explore
+
+<img src={EventPage} alt="Event Page" style={{ width: 720 }} />
+
+### New: Dancing hedgehogs
+
+Waiting for an insight to load? Our favorite mascot, Max, will now keep you company!
+
+<img src={DancingHog} alt="Dancing HedgeHog" style={{ width: 720 }} />
+
+### Improved: Save event columns
+
+You can now select and reorder the custom event columns that you want to see on the live events page. You can save the selected columns selected at the team level, so your colleagues can benefit too.
+<img src={ConfigurableColumns} alt="Configurable Columns" style={{ width: 400 }} />
+
+### Improved: Faster response times in the app
+
+We've made it faster to load insights and recordings. We realized that some data payloads were quite big, which resulted in the network slowing things down. To improve this, we added compression to a few of the endpoints in the app, and now see at least 50% smaller responses sent to clients.  
+
+<img src={APICompression} alt="a screenshot showing an API client receiving 308KB instead of 1.4MB of data" style={{ width: 800 }} />
+
+Here's to things being faster!
+
+### Other improvements & fixes
+
+Version 1.37 also adds hundreds of other improvements and fixes, including...
```

Comment:
I propose also mentioning https://github.com/PostHog/posthog/pull/10187, or more broadly:

> The retry policy of apps has been consolidated. Built-in retries with `RetryError` are now more widely available. Additionally, you will be notified by email if an app fails to load in your project due to a fatal `setupPlugin` error ([email configuration required](https://posthog.com/docs/self-host/configure/email)). See [Apps developer reference](https://posthog.com/docs/apps/build/reference#maximizing-reliability-with-retryerror) for more details.

Reply:
Nice! Added a section for this